File: submissions/sm_028_sagar/week_03/day_3/session_2/average_divisible_while.js
var number = 5; // for checking divisibility
var limit = 10;
var sum = 0;
var count = 0;
var i = 0;
while (i <= limit) {

    //checking divisibility 
    while (i % number == 0) {
        count++;
        sum = sum + i;
        console.log("Numbers", i)
        break;
    }
    i++;
}
console.log(sum);

//calculating average
var average = sum / count;

console.log("Average for the numbers divisible by " + number + " is:", average);


//TEST CASE
// input:
// number = 5;
// limit = 19;
// output:
// Average for the numbers divisible by 5 is: 5

File: submissions/sm_012_gaurav/week_13/day_4/session_1/average_divisible_while.py
limit = int(input("Enter Limit: "))
no = int(input("Enter Number: "))
i = 0
s = 0
cnt = 0
while i < limit + 1:
    if i%no == 0:
        cnt += 1
        s += i
    i += 1
print(s/cnt)
